test/manifests-version/deployment_version_test.go

```diff
@@ -55,7 +53,7 @@ func TestDeploymentVersions(t *testing.T) {
 		t.Errorf("Parsed object is not a Deployment object")
 	}
 
-	if deployment.Spec.Template.Labels["rht.comp_ver"] != version.ThreescaleRelease {
-		t.Errorf("rht.comp_ver differ: expected: %s; found: %s", version.ThreescaleRelease, deployment.Spec.Template.Labels["rht.comp_ver"])
-	}
+	//if deployment.Spec.Template.Labels["rht.comp_ver"] != version.ThreescaleRelease {
+	//	t.Errorf("rht.comp_ver differ: expected: %s; found: %s", version.ThreescaleRelease, deployment.Spec.Template.Labels["rht.comp_ver"])
+	//}
```

Comment:
I think we can possibly remove this section instead of commenting out.

Reply:
done. Thank you